modules/help_editor/js/help_editor_helper.js

```diff
@@ -35,7 +34,48 @@ $("input[name=preview]").click(function(e) {
      e.preventDefault();
 
 });
+$("#save-help").click(function(e) {
+    e.preventDefault();
+    var title   = $('input[name="title"]').val(),
+        content = $('textarea[name="content"]').val(),
+        section = $("#section").val(),
+        subsection = $("#subsection").val(),
+        parentID = $("#parentID").val(),
+        helpID = $("#helpID").val();
+
+    $.ajax({
+        type: 'POST',
+        url: loris.BaseURL + '/help_editor/ajax/process.php',
+        data: {
+            title: title ? title : '',
+            content: content ? content : '',
+            section: section ? section : '',
+            subsection: subsection ? subsection : '',
+            parentID: parentID ? parentID : '',
+            helpID: helpID ? helpID : '',
+        },
+        success: function() {
+            swal({
+                title: "Content update successful!",
+                type: "success",
+                confirmButtonText: "Return to " + title,
```

Comment:
If I edit New Profile, for example, this confirmation button says "Return to New Profile" but takes me back to the Help Editor.

This should be changed to say e.g. "Back to Help Editor" or perform the redirect

Reply:
Oops just saw you mentioned this to Xavier!

Reply:
done!